### ISSUE TITLE
Improve print styles for Design Manual

### ISSUE THREAD

**Issue by mebates** [cfpb/design-manual#250]
Users may want to print individual pages of the style guide for reference. See issue #217

**Comment by Dnpizarro** [comment]
10/20 Design Manual Review update:
- There was consensus around creating a print stylesheet to avoid having to update the printed version.
- Updates would then only need to be made to the Design Manual.
- The print stylesheet could be a great project for incoming fellows during immersion.

@nataliafitzgerald any thoughts or feedback regarding some of the proposals?

**Comment by nataliafitzgerald** [comment]
@Dnpizarro - I am in agreement that there is a need for a print style sheet and that we can continue to phase out the printed version of the Design Manual (as we have already done). @elizbond is this something we can add to the immersion projects list. Is this project of the right scale for that list?

**Comment by stephanieosan** [comment]
Do we still need/want print styles for the design manual? Do we still give new designers a printed copy of our style guide? Or do we just give them the link to the design manual?

This seems pretty low priority...

**Comment by caheberer** [comment]
Whatever is implemented for #281 should be implemented here. Waiting on an update.
